### PR TITLE
Generated  models and request builders

### PR DESCRIPTION
**Pull request by andrueastman**
This pull request was automatically created by Azure Pipelines. **Important** Check for unexpected deletions or changes in this PR.

### CODE REVIEW

**Review by baywet — CHANGES_REQUESTED** [review]
@andrueastman a number of things have been removed and I'm not sure whether they are expected:
1. /users : skip query param, expected
1. /groups/validateProperties? 
1. /groups/{id}/members cast to user, application, servicePrincipal, device, orgContact. I'm guessing cast to user and service principal should not have been removed?
1. /orgContacts/{id}/ cast to user, servicePrincipal, orgContact, application, device. I'm guessing this is expected.
1. /servicePrincipals/{id}/memberOf/ cast to user, application, servicePrincipal, device, orgContact. Guessing this is expected.
1. /users/{id}/memberOf/ cast to user, application, servicePrincipal, device, orgContact. Guessing this is expected.
1. /devices/{id}/memberOf/ cast to user, application, servicePrincipal, device, orgContact. Guessing this is expected.

File: src/main/java/com/microsoft/graph/requests/GroupCollectionRequestBuilder.java [hunk 1]
```diff
@@ -65,16 +64,6 @@ public GroupDeltaCollectionRequestBuilder delta() {
         return new GroupDeltaCollectionRequestBuilder(getRequestUrlWithAdditionalSegment("microsoft.graph.delta"), getClient(), null);
     }
 
-    /**
-     * Gets a builder to execute the method
-     * @return the request builder
-     * @param parameters the parameters for the service method
-     */
-    @Nonnull
-    public DirectoryObjectValidatePropertiesRequestBuilder validateProperties(@Nonnull final DirectoryObjectValidatePropertiesParameterSet parameters) {
```
Comment (andrueastman) [inline src/main/java/com/microsoft/graph/requests/GroupCollectionRequestBuilder.java:74]: This shouldn't be dropped.

File: src/main/java/com/microsoft/graph/requests/DeviceRequestBuilder.java [hunk 2]
```diff
@@ -82,26 +82,6 @@ public com.microsoft.graph.requests.DirectoryObjectCollectionWithReferencesReque
     public com.microsoft.graph.requests.DirectoryObjectWithReferenceRequestBuilder memberOf(@Nonnull final String id) {
         return new com.microsoft.graph.requests.DirectoryObjectWithReferenceRequestBuilder(getRequestUrlWithAdditionalSegment("memberOf") + "/" + id, getClient(), null);
     }
-    /**
-     *  Gets a request builder for the User collection
-     *
-     * @return the collection request builder
-     */
-    @Nonnull
-    public com.microsoft.graph.requests.UserCollectionRequestBuilder memberOfAsUser() {
```
Comment (andrueastman) [inline src/main/java/com/microsoft/graph/requests/DeviceRequestBuilder.java:91]: Related to fix at https://github.com/microsoftgraph/msgraph-metadata/pull/286

File: src/main/java/com/microsoft/graph/requests/UserCollectionRequest.java [hunk 3]
```diff
@@ -162,18 +162,6 @@ public UserCollectionRequest count() {
         addCountOption(true);
         return this;
     }
-    /**
-     * Sets the skip value for the request
-     *
-     * @param value of the number of items to skip
```
Comment (andrueastman) [inline src/main/java/com/microsoft/graph/requests/UserCollectionRequest.java:168]: Related to fix at https://github.com/microsoftgraph/msgraph-metadata/pull/283

**Comment by andrueastman** [comment]
> /groups/{id}/members cast to user, application, servicePrincipal, device, orgContact. I'm guessing cast to user and service principal should not have been removed?

I believe this is removing the `memberOf` casts for a group because a group can only be a member of another group.

**Comment by andrueastman** [comment]
Closing this for now as it superceded by https://github.com/microsoftgraph/msgraph-sdk-java/pull/1371

**Pull request closed without merging**